numpyro/distributions/transforms.py

```diff
@@ -1121,6 +1121,19 @@ def _transform_to_interval(constraint):
     )
 
 
+@biject_to.register(constraints.open_interval)
```

Comment:
You can add this to the above `def _transform_to_interval(constraint):`, so no need to reimplement the transform.